src/partials/20_section-hot-deals.html

```diff
@@ -0,0 +1,79 @@
+<div class="hot-deals">
+  <div class="container">
+    <div class="hot-deals-slider">
+      <div class="slider-item">
+        <div class="row">
+          <div class="case col-4">
+            <div class="content">
+              <div class="cart">
+                <div class="cart-title">
+                  <p>hot deals</p>
+                  <div class="col-auto dots">
+                    <ul>
+                      <li><a href="#" class="active"></a></li>
+                      <li><a href="#"></a></li>
+                      <li><a href="#"></a></li>
+                    </ul>
+                  </div>
+                </div>
+                <div class="cart-body">
+                  <img class="product-photo" src="./images/hot-deals.png" alt="" />
+                  <div class="add-to-cart">
+                    <i class="fa fa-shopping-basket"></i>
+                    <p>add to cart</p>
+                  </div>
+                  <div class="time-count">
+                    <div class="days time"><span class="count">25</span>days</div>
+                    <div class="hours time"><span class="count">10</span>hrs</div>
+                    <div class="minuts time"><span class="count">45</span>mins</div>
+                    <div class="seconds time"><span class="count">30</span>secs</div>
+                  </div>
+                </div>
+                <div class="cart-footer">
+                  <div class="product-name"><p>Aenean Ru Bristique</p></div>
+                  <div class="stars">
+                    <a href="#" class="full"></a> <a href="#" class="full"></a>
+                    <a href="#"></a> <a href="#"></a> <a href="#"></a>
+                  </div>
+                  <div class="pre-footer-cart">
+                    <ul class="actions">
+                      <div class="outlines">
+                        <a href="#" class="btn-outline"><i class="far fa-eye"></i></a>
+                        <a href="#" class="btn-outline"><i class="far fa-heart"></i></a>
+                        <a href="#" class="btn-outline"
+                          ><i class="fas fa-exchange-alt"></i
+                        ></a>
+                      </div>
+                    </ul>
+                    <div class="price">
+                      <span class="old-price">$350.00</span>
+                      <span class="current-price">$300.00</span>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="case col-8">
+            <div class="content">
+              <div class="slide-img-desc">
+                <img src="./images/indoor-furniture.jpg" alt="" />
```

Comment:
Wydaje mi się że zdjęcie powinno być "schowane" pod górne menu, tak żeby cień z górnego menu zachodził na zdjęcie. Obecnie zdjęcie jest "nad" cieniem z górnego menu. (nigdy nie lubiłem box-shadow - sporo z tym problemów zawsze ;) )
![review](https://user-images.githubusercontent.com/49732790/63323502-3f51a680-c326-11e9-8e12-94a9f5717d82.jpeg)

Reply:
mówisz - masz :) do 30 minut temu nie miałem pojęcia o co chodzi. okazuje się, że to dość proste - sekcja Hot Deals "wychodzi spod" nagłówka w tej chwili. mam nową sprawność :D

Reply:
No i git !